README.md

```diff
@@ -112,37 +114,22 @@ This repo can be used as an isolated tool to build and test your integration wit
 </script>
 ```
 
-## Configuration
-Tailor web chat to your needs by initializing it with your own custom options. The web chat configuration options are defined by the parameters of watsonAssistantChatOptions.
+## Types of integrations
 
-In addition to the parameters listed [here](https://web-chat.global.assistant.watson.cloud.ibm.com/docs.html?to=api-configuration#configurationobject), it supports the following options:
+There are two basic types of integrations.
 
-#### serviceDesk
-```
-serviceDesk: {
-  availabilityTimeoutSeconds: 30,
-}
-```
-
-- `availabilityTimeoutSeconds`: The timeout value in seconds to use when determining agent availability. When a connect_to_agent response is received, the system will ask the service desk if any agents are available. If no response is received within the timeout window, the system will return "false" to indicate no agents are available. The default in web chat is 5 seconds.
+1. A seamless integration with the existing web chat UI. The user will use the web chat input field to send messages to an agent and messages from an agent will be displayed in the same message list along with messages from Watson Assistant. These integrations require fully implementing the service desk integration API provided by web chat.
+2. A custom panel implementation embedding a 3rd party chat widget. The user will switch to a different panel where they will see a different UI created by a 3rd party. A different field from the standard web chat input field will generally be available and messages to and from the agent will be displayed in a separate view and not part of the messages from Watson Assistant. These integrations generally require a lot less code and only require embedding another existing widget into web chat.
```

Comment:
`These integrations generally require a lot less code and only require embedding another existing widget into web chat`, `but can suffer from a degraded UX experience` or something like that?